src/views/Project/index.jsx

```diff
@@ -36,6 +41,31 @@ const tagReposMapping = repositories =>
     },
   }),
 })
+@withStyles(theme => ({
+  root: {
+    background: theme.palette.background,
+  },
+  header: {
+    height: 60,
+    paddingBottom: theme.spacing.unit,
+  },
+  container: {
+    marginTop: 60,
+    paddingTop: theme.spacing.unit,
+    minHeight: `calc(100vh - 180px - ${3 * theme.spacing.unit}px)`,
+  },
+  link: {
+    textDecoration: 'none',
+    position: 'absolute',
+    alignItems: 'center',
+    '& svg': {
+      fill: theme.palette.common.white,
+    },
+  },
+  title: {
+    marginLeft: 41,
```

Comment:
We don't need this because the back button is styled with position `absolute` and so the title will automatically take the full row width.

Reply:
i use it to avoid the title overlap with back button 
<img width="1730" alt="screen shot 2018-06-27 at 1 43 53 am" src="https://user-images.githubusercontent.com/24544912/41929779-bd9f0700-79ab-11e8-8c1e-9e8e8c946eb2.png">

Reply:
If you **pick** a mobile device and then refresh, then it should work. This is how it looks on mine when I select Nokia Lumia 520.

![screen shot 2018-06-26 at 2 18 09 pm](https://user-images.githubusercontent.com/3766511/41931397-f2fd3418-794b-11e8-80e4-fffa8529b131.png)

Reply:
@helfi92 have u tried this url? http://localhost:5000/projects/contentservices i picked a same device (Nokia Lumia) but still got something like this 
<img width="1730" alt="screen shot 2018-06-27 at 2 32 46 am" src="https://user-images.githubusercontent.com/24544912/41932046-78257112-79b2-11e8-8a1d-bdc5390545de.png">

Reply:
Still works. Did you try refreshing the page?

![screen shot 2018-06-26 at 2 37 50 pm](https://user-images.githubusercontent.com/3766511/41932288-9a286ff8-794e-11e8-9420-4567ff0ce8c8.png)

![screen shot 2018-06-26 at 2 35 45 pm](https://user-images.githubusercontent.com/3766511/41932301-a74c2422-794e-11e8-8d73-13cbc0c4c330.png)

Reply:
It still happen when i remove the marginLeft

Reply:
Have you tried to see if it works on Chrome? With the marginLeft applied, the title is no longer centered and is shifted to the right:

![screen shot 2018-06-27 at 9 11 47 am](https://user-images.githubusercontent.com/3766511/41976133-4042a688-79ea-11e8-936e-91c11d52f6fc.png)